packages/core/examples/multiVolumeAPI/index.ts

```diff
@@ -116,11 +116,13 @@ addButtonToToolbar({
     } else {
       // Add the PET volume to the viewport. It is in the same DICOM Frame Of Reference/worldspace
       // If it was in a different frame of reference, you would need to register it first.
+      const slabThickness = 0.1;
```

Comment:
Is this required? if always required can we bake it in the addVolumes? e.g., if not defined use 0.1? Also 1-2 lines of explanation for this 0.1 is good

Reply:
no it is not required. I will remove it. If it will be undefined, when setting the clipping planes in resetCamera it will be used the default value (0.05)